virttest/utils_libvirt/libvirt_misc.py

```diff
@@ -0,0 +1,28 @@
+"""
+Virsh command related utility functions which do not belong to special
```

Comment:
Better add copyright declare for newly created file

Reply:
updated